doc/api/unicode.md

```diff
@@ -0,0 +1,55 @@
+# icu
+
+    Stability: 1 - Experimental
+
+The `unicode` module provides an interface to Unicode and Internationalization
+functionality provided by the ICU4C library.
+
+```js
+const unicode = require('unicode');
+```
+
+## unicode.transcode(buf, from_enc, to_enc)
+
+* `buf` {Buffer} A `Buffer` instance
+* `from_enc` {string} The current encoding
+* `to_enc` {string} The target encoding
+
+Re-encodes the given `Buffer` from one character encoding to another. Returns
+a new `Buffer` instance.
```

Comment:
This description here is not accurate, [when `from_enc` and `to_enc` is same](https://github.com/jasnell/node/blob/842e0810de3ebdba739695c259dc9b8c8a58f840/lib/unicode.js#L70-L71), this function just returns the original `source` instance, not a new instance of `Buffer`.

_Suggestion: remove the "new" adjective._